### PR TITLE
Fix RHEL 7 & 8 based distros to work around /tmp execution issues

### LINKED ISSUE #155: Files in /tmp Cannot be Executed in Centos 7
By default, on Centos 7, files in `/tmp` cannot be executed (even if chmod 777) because the filesystem mount prevents it. The result is that all attempts to install RVM fail:
```
TASK [rvm_io.ruby : Install rvm] ******************************************************************************************************************************************************************************************************************
fatal: [myserver]: FAILED! => {"changed": false, "cmd": "/tmp/rvm-installer.sh stable --path /root/.rvm --auto-dotfiles --user-install", "failed": true, "msg": "[Errno 13] Permission denied", "rc": 13}
```

The solution is thankfully pretty simple - instead of running `/tmp/something`, just run `/bin/bash /tmp/something`. Therefore, in tasks/rvm.yml, the task to install RVM becomes:
```
- name: Install rvm
  command: >
    /bin/bash {{ rvm1_temp_download_path }}/rvm-installer.sh {{ rvm1_rvm_version }}
    --path {{ rvm1_install_path }} {{ rvm1_install_flags }}
  when: not rvm_binary.stat.exists
```
Incidentally, using `shell` instead of `command` doesn't work either. As far as I can see, this minor change will be compatible with other distributions and versions though, so hopefully low-risk.

### PR DESCRIPTION
**Pull request by stevenhaddox**
Implements the suggested fix from #155 in order to enable rvm to install properly on CentOS 8 (and hopefully 7 and a few other distro variations as well).

Fixes #155

### CODE REVIEW

**Comment by stevenhaddox** [comment]
Shout out to @coofercat for the suggested fix.

**Review by danochoa — APPROVED** [review]
lgtm

**Comment by nartz** [comment]
I could use this as well - thanks!

**Comment by stevenhaddox** [comment]
Thought I'd follow-up on this to see if it might be able to be merged (I came across my #TODO in my galaxy file preventing me from being able to enable the galaxy collection for rvm until this is merged in).  
cc @danochoa, maybe?

File: tasks/rvm.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
 - name: Install rvm
   shell: >
-    {{ rvm1_temp_download_path }}/rvm-installer.sh {{ rvm1_rvm_version }}
+    /bin/bash {{ rvm1_temp_download_path }}/rvm-installer.sh {{ rvm1_rvm_version }}
```
Comment (sfgeorge) [inline tasks/rvm.yml:47]: Tweak this to `/usr/bin/env bash` for maximum portability, and then LGTM 👍

**Comment by sfgeorge** [comment]
@stevenhaddox Fantastic, thank you for the quick change.  I'm terribly sorry to ping-pong with another request, but could you add a brief entry in CHANGELOG.md?  Just a bullet would about fixing installation error in CentOS 7&8 would do.

**Comment by stevenhaddox** [comment]
@sfgeorge no need to apologize. Thanks for your patience. Hopefully that last push is what you're looking for 🤞🏻

**Comment by sfgeorge** [comment]
I shall bother you no more, this hungry commit-monster is finally satiated!

Thank you for the great fix, @stevenhaddox !

**Review by sfgeorge — APPROVED** [review]

**Comment by sfgeorge** [comment]
Oops, I don't have ability to move this forward.  @danochoa Can you merge if you deem it ready?

**Review by danochoa — APPROVED** [review]

**Comment by danochoa** [comment]
It looks like the old CI server is no longer working. Maybe @pkuczynski can help? I think I don't have access to fix this but I can poke around a bit later. https://travis-ci.org/github/rvm/rvm1-ansible/requests

**Comment by sfgeorge** [comment]
@danochoa I noticed the same and tracking as #233.

tl;dr With new pricing changes, it's possible but semi-painful to get a renewal of some Travis-CI build hours going. Alternatively, we can move to GitHub Actions.

**Comment by pkuczynski** [comment]
Thanks!
